colorutils.cpp

```diff
@@ -1117,18 +1117,18 @@ void nblendPaletteTowardPalette( CRGBPalette16& current, CRGBPalette16& target,
     p2 = (uint8_t*)target.entries;
 
     const uint8_t totalChannels = sizeof(CRGBPalette16);
-    for( uint8_t i = 0; i < totalChannels; i++) {
+    for( uint8_t i = 0; i < totalChannels; ++i) {
         // if the values are equal, no changes are needed
         if( p1[i] == p2[i] ) { continue; }
 
         // if the current value is less than the target, increase it by one
-        if( p1[i] < p2[i] ) { p1[i]++; changes++; }
+        if( p1[i] < p2[i] ) { ++p1[i]; ++changes; }
```

Comment:
This threw me, because the suffix increment/decrement operators have the [same](https://en.cppreference.com/w/cpp/language/operator_precedence) precedence as the array subscript operator. 
Then I saw that **_prefix_** increment / decrement operators are separately listed, with lower precedence.  So, this is actually OK.  :)

Reply:
Actually had to look that up too when I did this